raiden/network/protocol.py

```diff
@@ -349,17 +354,22 @@ def receive(self, data):
                 if log.isEnabledFor(logging.WARN):
                     log.warn(str(e))
                 return
+            except:
+                log.exception('unexpected exception raised.')
```

Comment:
nah, ERROR level is good. I meant for the exceptions above that, https://github.com/raiden-network/raiden/pull/413/files/27e0fe4e39bbcb537d6a478a4a680bd99be004a3#diff-36a2cc663db8263416e47081d1939e5eR350 a debug message, to trace the code path if necessary